releases.json

```diff
@@ -2708,7 +2708,8 @@
     ],
     "versions": [
       "0.6.0-1",
-      "0.5.0-1"
+      "0.5.0-1",
+      "fd2263bbeb7c6d330fbfaf556fa1997aa1091179-1"
```

Comment:
Based off of the CI failures you might need to put this first in the list instead of at the end